File: kodovi/PosiljkaApp_video6c/app/src/main/java/android/fit/ba/posiljka/data/OpstinaPregledVM.java
package android.fit.ba.posiljka.data;

import java.io.Serializable;
import java.util.List;

public class OpstinaPregledVM implements Serializable
    {
       public static class Row implements Serializable
       {
           public int id ;
           public String naziv;
           public String drzava ;
       }

       public List<Row> rows;

    }

File: PosiljkaWebApi_video5_video6/Posiljka.Web/ViewModels/api/OpstinaPregledVM.cs
using System;
using System.Collections.Generic;

namespace Posiljka.Web.ViewModels.api
{
        
    public class OpstinaPregledVM
    {
        public class Row
        {
            public int id;
            public String naziv;
            public String drzava;
        }

        public List<Row> rows;

    }
}
